gatorgrade/output/output_functions.py

```diff
@@ -1,7 +1,29 @@
-"""Add docstring."""
+"""This module is used for storing the main functions requested from the Github Issue Tracker for the output team.
+For instance, functions dealing with percentage output, description output, and colorization of text."""
 
+import gator
+import os
+from output import output_tools
+from typing import List, Tuple
 
-# This class is used for storing the main functions requested
-# from the Github Issue Tracker for the output team.
-# For instance, functions dealing with percentage output,
-# description output, and colorization of text.
+# Commands are received as tuple of lists with lists
+# The tuple is used for future upgradeability for when shell commands are introduced
+# The string is the path to the file or filename and the list is the commands for the file.

```

Comment:
It would make this more readable and ensure that this documentation makes it to the website if you include this in the docstring of this function. What do you think about that?

Reply:
I also think it would be super helpful if you can provide an example in the comments of how the input to this function is expected to look like